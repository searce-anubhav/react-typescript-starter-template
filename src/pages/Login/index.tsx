import * as React from 'react';
import Button from 'components/Button';
import Input from 'components/Input';
import { Link } from 'react-router-dom';

const Login: React.FC = (): JSX.Element => {
	return (
		<React.Fragment>
			<div className='level m-6'>
				<Link to='/'>
					<Button className='button is-warning m-1'>Go to Home</Button>
				</Link>
				<Input
					className='input is-warning m-1'
					type='text'
					placeholder='Enter Name'
				/>
			</div>
		</React.Fragment>
	);
};

export default Login;
