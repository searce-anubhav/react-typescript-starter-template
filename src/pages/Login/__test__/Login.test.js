import React from 'react';
import renderer from 'react-test-renderer';
import Login from 'pages/Login';
import { MemoryRouter } from 'react-router-dom';

test('Render Login Snapshot and Match', () => {
	const component = renderer.create(
		<MemoryRouter>
			<Login></Login>
		</MemoryRouter>
	);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
