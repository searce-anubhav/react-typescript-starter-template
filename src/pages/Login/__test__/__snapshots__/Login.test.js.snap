// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Login Snapshot and Match 1`] = `
<a
  href="/"
  onClick={[Function]}
>
  <button
    className="chakra-button css-5c2176"
    type="button"
  >
    Go to Home
  </button>
</a>
`;
